src/utils/spaceBetween.ts

```diff
@@ -62,13 +62,15 @@ export const spaceBetween = (
     .wrap(left, {
       width: leftTextWidth,
       break: true,
+      noTrim: true,
     })
     .split('\n');
 
   const rightWrappedTextArray: string[] = wordwrap
     .wrap(right, {
       width: rightTextWidth,
       break: true,
+      noTrim: true,
```

Comment:
First of all thanks a lot for this PR!

Small comment. Could you please also cover this case in the test file?
https://github.com/tr3v3r/react-native-esc-pos-printer/blob/main/src/utils/__tests__/spaceBetween.test.tsx

Thanks

Reply:
Sure will do shortly

Reply:
I have pushed a fix that fixes the tests. I think it's better approach to allow trimming of the text if the text is indeed larger than space available. This way current tests pass without modifying test file as well. Tested this also on my local project, works great.

Reply:
@tr3v3r keen to hear your thoughts :)

Reply:
@tolypash Thanks one more time for the update!

What I'm thinking is to extend a bit this logic to a more generic one. I'll try to prepare and improve soon.

  For now mI'd propose to go with the new param `noTrim` in `spaceBetween`:

```typescript

export const spaceBetween = (
  length: number,
  {
    left,
    right,
    textToWrap = 'left',
    textToWrapWidth,
    gapSymbol = ' ',
    noTrim = false
  }: ISpaceBetweenParams
......

  const leftWrappedTextArray: string[] = wordwrap
    .wrap(left, {
      width: leftTextWidth,
      break: true,
      noTrim
    })
    .split('\n');

```

Something like this. What do you think? Will it work for you?

Reply:
I think this will work for sure yes! I will try make it work in this PR

Reply:
Just pushed it. Tests pass as well

Reply:
Thanks! Will prepare new release today